packages/cloud_firestore/test/cloud_firestore_test.dart

```diff
@@ -64,6 +64,14 @@ void main() {
             return handle;
           case 'DocumentReference#setData':
             return true;
+          case 'DocumentReference#get':
+            if (methodCall.arguments['path'] == 'foo/bar') {
+              return <String, dynamic>{
+                'path': 'foo/bar',
+                'data': <String, dynamic>{'key1': 'val1'}
+              };
+            }
+            return new ArgumentError('Unknown database path');
```

Comment:
I would think this would throw a PlatformException to more closely mimic the native behavior.

Reply:
Done!